src/lev-dashboard/DashboardCounter.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const StyledCounter = styled.div`
```

Comment:
I think this should be called StyledDiv as it isn't a counter at this point.

Reply:
Agreed

Reply:
(You could also style at the end if you preferred. i.e. Create an unstyled counter and then style it.)

Reply:
Changed name. Have left styling as is.